packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_MeshDatabase.hpp

```diff
@@ -293,11 +297,14 @@ MeshDatabase::MeshDatabase(Teuchos::RCP<const Teuchos::Comm<int> > comm,
     }
   }
 
-  auto _ghostNodeGlobalIDs = ghostNodeGlobalIDs_.getHostView(Tpetra::Access::ReadWrite);
-  Kokkos::resize(_ghostNodeGlobalIDs,my_ghost_nodes.size());
-  for(auto k=my_ghost_nodes.begin(); k!=my_ghost_nodes.end(); k++) {
-    size_t kk = std::distance(my_ghost_nodes.begin(),k);
+  ghostNodeGlobalIDs_ = global_ordinal_view_type(globalDualViewType("ghostNodeGlobalIDs_",my_ghost_nodes.size()));
+  {
+    auto _ghostNodeGlobalIDs = ghostNodeGlobalIDs_.getHostView(Tpetra::Access::ReadWrite);
+    Kokkos::resize(_ghostNodeGlobalIDs,my_ghost_nodes.size());
```

Comment:
This resize can go away since ghostNodeGlobalIDs_ was just constructed to be that size.

Reply:
Good catch!